pkg/worker/types.go

```diff
@@ -0,0 +1,21 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package worker
+
+import (
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/events"
+)
+
+// EventProcessor provides a mechanism to act on events in the internal queue.
```

Comment:
I would have expected to see this deleted from https://github.com/Azure/application-gateway-kubernetes-ingress/blob/fec83cb83c52385afafebb2998825ab163b503d0/pkg/eventqueue/types.go#L14-L17

But perhaps you are getting to this eventually?

Reply:
yes, that will go away with the package.